src/index.tsx

```diff
@@ -7,7 +7,7 @@ import { BrowserRouter } from 'react-router-dom';
 import { ApolloProvider, ApolloClient, InMemoryCache } from "@apollo/client";
 
 const client = new ApolloClient({
-  uri: "https://6fd168b2-d698-498e-91e5-29866384a6bd.mock.pstmn.io/graphql",
+  uri: "https://bookworm-be.herokuapp.com/graphql",
```

Comment:
Are you guys using environment variables? It's usually worth storing URLs within your .env. But not neccessary.

Reply:
Not entirely sure but I will look into this and find out. Thank you for that suggestion!

Reply:
Agreed! It's great to get in the habit of storing URLs in a .env for good security hygiene. Check out this article if you've never done this before: https://adostes.medium.com/using-environment-variables-in-a-react-application-ac3b6c307373